source/aggregation-pipeline-builder.txt

```diff
@@ -370,6 +374,35 @@ To create a view from your pipeline results:
 
 .. include:: /includes/fact-create-view-from-pipeline.rst
 
+.. _refresh-document-counts-after-deletes:
+
+Refreshing Document Counts
+--------------------------
+
+After a document delete or add operation the :guilabel:`count results` 
```

Comment:
I would consider rewording this sentence so it's clearer *why* the user needs to perform this action:

"When you delete or add a document, the :guilabel:`count results` value does not automatically update to reflect the new document count."